.github/workflows/release.yml

```diff
@@ -9,7 +9,7 @@ on:
       - '*'
 
 env:
-  GO_VERSION: '1.18'
+  GO_VERSION: '1.20.4'
```

Comment:
Suppose, the version in `go.mod` and `Dockerfile` must be updated also.

Reply:
Discussed the problem and decided not to change the versions in go.mod and Dockerfile for now because they don't affect the vulnerabilities in the delivered binary.